test/models/contact_test.rb

```diff
@@ -0,0 +1,42 @@
+require 'test_helper'
+
+class ContactTest < ActiveSupport::TestCase
+  def setup
```

Comment:
Technically nothing can go wrong here, but You'd probably want to either include `super` or convert this to a block.